docs/source/manual/user-manual.rst

```diff
@@ -437,6 +437,18 @@ L4: Intensive scan
 
 The premise of the test profile is to verify whether an attacker can exploit vulnerabilities to give himself more extensive access to the tested environment. Thus, known exploit code is applied in this level.
 
+
+Object origin types
+-------------------
+Each object in OpenKAT has an origin type. The origin type defines how the object was added to the database and depending on the origin type, this also affects the garbage collection of that object. There are 4 different origin types:
```

Comment:
Perhaps nitpicky, but each object in OpenKAT has an _Origin_, and every Origin is of one of these _OriginTypes_. But not sure if this is already how we talk about Origin types in the UI